capstone/src/main/webapp/form-loader.js

```diff
@@ -0,0 +1,113 @@
+var CLIENT_ID = '1002281613740-gsmtp3tteqgtoformikuglhu8fr388et.apps.googleusercontent.com';
```

Comment:
Is this really a form-loader or a thing for handling authentication? These concepts should probably be separated out into separate files, so that any page can re-use the auth part easily event if they dont want the forms bit attached to it.

Reply:
Yeah, sounds good to make another JS file to separate the forms and authentication functionalities.